### ISSUE TITLE
EfficientDet-D3-4(with Weight)

### ISSUE THREAD

**Issue by Jilliansea** [toandaominh1997/EfficientDet.Pytorch#133]
Where can I found EfficientDet-D3-D7(with Weight)? Thank you

**Comment by EdwardMessi** [comment]
I also want to know..It's so difficult to train this all by myself.

**Comment by zhui-ying** [comment]
Can I get your pre-train model?

**Comment by tadbeer** [comment]
Hi @EdwardMessi , @Jilliansea  I am trying to train this on my own dataset, but cant achieve a successfull training, that is, the MAP is not increasing with increasing epochs. Have you tried training on your own dataset? Could you kindly get in touch with me?

**Comment by zylo117** [comment]
> Hi @EdwardMessi , @Jilliansea I am trying to train this on my own dataset, but cant achieve a successfull training, that is, the MAP is not increasing with increasing epochs. Have you tried training on your own dataset? Could you kindly get in touch with me?

the backbone is altered in the wrong way, the conv stride of efficientnet should not be changed.

for more info, you can checkout my repo,

https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch

**Comment by tadbeer** [comment]
Thanks @zylo117 . So can i go ahead and try finetuning on my own data using your repository? Has anyone or you yourself have, reported a successfull finetuning on a self made dataset?

**Comment by zylo117** [comment]
> Thanks @zylo117 . So can i go ahead and try finetuning on my own data using your repository? Has anyone or you yourself have, reported a successfull finetuning on a self made dataset?

of course, the training script is still testing and will be released within a few days. and yes, i trained on a custom dataset, it worked alright.
